tasks/cat3.yml

```diff
@@ -1,4 +1,893 @@
 ---
+# CAT III findings
+
+- name: "| LOW | V-38437 | AUDIT |\n
+        Automated file system mounting tools must not be enabled unless needed."
+  command: chkconfig 'autofs' --list
+  register: autofs_service_audit
+  changed_when: false
+  failed_when: false
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38437
+      - autofs
+      - services
+      - audit
+
+- name: "| LOW | V-38437 | PATCH |\n
+        Automated file system mounting tools must not be enabled unless needed."
+  service:
+      name: autofs
+      state: stopped
+      enabled: no
+  when: "':on' in autofs_service_audit.stdout"
+  tags:
+      - cat3
+      - low
+      - V-38437
+      - autofs
+      - services
+      - patch
+
+# V-38438 grub
+
+# V-38447 rpm hashes
+- name: "| LOW | V-38447, V-38452, V-38453, V-38454 | AUDIT |\n
+        Verify all rpm packages and store output for later use."
+  command: rpm -Va
+  changed_when: false
+  failed_when: false
+  register: rpm_verify_packages
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38447
+      - V-38452
+      - V-38453
+      - V-38454
+      - audit
+
+- name: "| LOW | V-38447 | AUDIT |\n
+        The system package management tool must verify contents of all files associated with packages."
+  command: rpm -qf {{ item | regex_replace('^..5......\\s+(/.+)$', '\\1') }}
+  register: rpm_integrity_audit
+  when: item | match("^..5......[^c]*/.+$")
+  with_items: rpm_verify_packages.stdout_lines
+  always_run: yes
+  tags:
+      - cat3
+      - low
+      - V-38447
+      - rpm
+      - audit
+
+- name: "| LOW | V-38447 | PATCH |\n
+        The system package management tool must verify contents of all files associated with packages."
+  command: yum reinstall -y {{ item.stdout }}
+  when: item|changed
+  with_items: rpm_integrity_audit.results
+  tags:
+      - cat3
+      - low
+      - V-38447
+      - rpm
+      - patch
+
+# V-38452 rpm system permissions
+- name: "| LOW | V-38452 | AUDIT |\n
+        The system package management tool must verify permissions on all files and directories associated with packages."
+  command: rpm -qf {{ item | regex_replace('^.M.......\\s+[a-z]?\\s*(/.+)$', '\\1') }}
```

Comment:
Gotcha. I still want to bring that up to the mailing list, for now, we'll leave this as is. I don't want to bikeshed this refactor. We can do some minor releases down the lane for optimization/beautification.